### ISSUE TITLE
linter: `import/namespace` inconsistently reports named imports

### ISSUE THREAD

**Issue by aleksanderkatan** [oxc-project/oxc#24073]
### What version of Oxlint are you using?

1.72.0

### What command did you run?

`oxlint`

### What does your `.oxlintrc.json` (or `oxlint.config.ts`) config file look like?

```ts
import { defineConfig } from 'oxlint';

export default defineConfig({
  plugins: ['import'],
});
```

### What happened?

Repro: [link](https://github.com/aleksanderkatan/oxlint-repro)

```ts
import { lib } from './module.ts';

console.log(lib['~unstable']); // reported
```
`lib['~unstable']` is reported by `import(namespace)` (Unable to validate computed reference to imported namespace "lib"). This report is silenced by an additional import line: 
```ts
import { otherExport } from './module.ts'; console.log(otherExport);
import { lib } from './module.ts';

console.log(lib['~unstable']); // not reported
```

Unless I misconfigured and misunderstood `eslint`, nothing should be reported in both cases, as `lib` is a named import, not a namespace import.

**Comment by camc314** [comment]
Opened a fix in #24094.

Root cause: `import/namespace` was iterating over import entries, but several non-actionable cases used `return` from `run_once` instead of skipping just that import entry. So when a regular named import like `otherExport` appeared before `lib`, the rule exited early and never checked the later `lib` import. Without that preceding import, `lib` was still checked, which caused the inconsistent behavior.

The fix changes those per-import early exits to `continue`, so ordinary named/default/unresolved imports are skipped without preventing later namespace imports from being validated. This intentionally keeps oxlint's stricter behavior for `export * as lib from './lib'`: `lib` is treated as a namespace object, so `lib["~unstable"]` still reports unless `allowComputed` is enabled.

Added regression coverage for both forms:

```ts
import { lib } from './module';
console.log(lib["~unstable"]);
```

and:

```ts
import { otherExport } from './module'; console.log(otherExport);
import { lib } from './module';
console.log(lib["~unstable"]);
```

Both now report consistently.
